dsc/tests/dsc_config_get.tests.ps1

```diff
@@ -27,7 +27,8 @@ Describe 'dsc config get tests' {
     It 'will fail if resource schema does not match' -Skip:(!$IsWindows) {
         $jsonPath = Join-Path $PSScriptRoot '../examples/invalid_schema.dsc.yaml'
         $config = Get-Content $jsonPath -Raw
-        $null = $config | dsc config get | ConvertFrom-Json
+        $testError = & {$config | dsc config get get 2>&1}
+        $testError | Select-String '^error:' -Quiet | Should -BeTrue
```

Comment:
If you're just comparing the output, it's better to use something like:

```powershell
$testError | Should -match '^error:'
```

As when it fails, you'll see both the expected and actual vs true/false.

Reply:
fixed

Reply:
tested. unfortunately, Pester -match operation does not seem to work in this pattern.